### PR TITLE
Delete .env.example

### PR DESCRIPTION
**Pull request by yusufdurr**

### CODE REVIEW

File: composer.lock
Comment (cekapykp) [inline composer.lock:1]: You should not send this file, delete it now, after mr you can upload it again with "composer update" @yusufdurr

File: .env.example
Comment (cekapykp) [inline .env.example:1]: Please do not remove this file, it is a guide for those building the next project. @yusufdurr

**Review by cekapykp — CHANGES_REQUESTED** [review]
2 PENDING COMMENTS

File: composer.lock
Comment (yusufdurr) [reply composer.lock:1]: Oh right man, oh right. @cekapykp

File: .env.example
Comment (yusufdurr) [reply .env.example:1]: Oo. I get it.

**Review by cekapykp — APPROVED** [review]

**Pull request merged**